src/main/modules/helmet/index.ts

```diff
@@ -19,6 +19,7 @@ export class Helmet {
 
   enableFor (app: express.Express) {
     app.use(helmet())
+    app.all(/^.+(?!\/public\/.*)$/, helmet.noCache())
```

Comment:
I suggest `^\/(?!public).*$`. Your regex matches `/public/stylesheets/application.css` which means that that file would not get cached.